xinference/deploy/docker/docker-compose-distributed.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.8'
+
+services:
+  xinference: &xinference
+    image: xprobe/xinference:v0.9.0
```

Comment:
Use xprobe/xinference:latest for every release.

Reply:
ok, changed to latest